aws-postgresql.yml

```diff
@@ -29,6 +29,7 @@ plans:
   display_name: "Subsume (Beta)"
   properties:
     subsume: true
+    storage_type: "gp2"
```

Comment:
Since this is a subsume plan that storage type should be picked up by the import and not assigned here.

Reply:
Yeah that's fair - this was just keeping it consistent with the previous subsume plan. I can remove this.